package.json

```diff
@@ -49,42 +49,37 @@
     "test": "yarn jest"
   },
   "dependencies": {
-    "@types/node-jose": "^1.1.5",
     "basic-auth": "^2.0.1",
-    "body-parser": "^1.19.0",
     "cors": "^2.8.5",
     "express": "^4.17.1",
-    "jsonwebtoken": "^8.5.1",
+    "jose": "^3.14.4",
     "lodash.isplainobject": "^4.0.6",
-    "node-jose": "^2.0.0",
     "uuid": "^8.3.2"
   },
   "devDependencies": {
     "@types/basic-auth": "^1.1.3",
-    "@types/body-parser": "^1.19.0",
-    "@types/cors": "^2.8.9",
-    "@types/express": "^4.17.9",
-    "@types/jest": "^26.0.19",
-    "@types/jsonwebtoken": "^8.5.0",
+    "@types/cors": "^2.8.12",
+    "@types/express": "^4.17.13",
+    "@types/jest": "^27.0.1",
     "@types/lodash.isplainobject": "^4.0.6",
-    "@types/node": "^16.4.10",
-    "@types/supertest": "^2.0.10",
-    "@types/uuid": "^8.3.0",
-    "@typescript-eslint/eslint-plugin": "^4.29.0",
-    "@typescript-eslint/parser": "^4.29.0",
-    "eslint": "^7.16.0",
+    "@types/node": "^12.20.19",
```

Comment:
As Node 12 is the lowest supported version, we should stick with type definitions for this version (ie. 12.x.y)